config/locales/welcome_email/de.yml

```diff
@@ -0,0 +1,25 @@
+en:

```

Comment:
The root needs to match to the language code, so here it should be `de:` in es-ES it should be `es-ES:` etc.